pkg/apis/serving/v1beta1/inference_service.go

```diff
@@ -78,6 +78,8 @@ type Batcher struct {
 
 // InferenceService is the Schema for the InferenceServices API
 // +k8s:openapi-gen=true
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +genclient
```

Comment:
Is this required for the client go generation ?

Reply:
according to https://kubebuilder.io/migration/legacy/migration_guide_v1tov2.html#migrate-the-apis, we do not need these markers anymore.